scripts/update-sources.ts

```diff
@@ -316,37 +319,34 @@ async function buildIosZano(platform: IosPlatform): Promise<void> {
     'install'
   ])
 
-  // Explode Zano archives and gather the objects:
-  async function unpackLib(libPath: string, name: string): Promise<void> {
-    console.log(`Unpacking lib${name}.a`)
-    const outPath = join(working, `unpack/${name}`)
-    await rm(outPath, { recursive: true, force: true })
-    await mkdir(outPath, { recursive: true })
-    await loudExec('ar', ['-x', libPath], { cwd: outPath })
+  // Link everything together into a single giant .o file:
+  console.log(`Linking zano-module.o for ${sdk} ${arch}`)
+  const objectPath = join(working, 'zano-module.o')
+  await loudExec(ld, [
+    '-r',
+    '-o',
+    objectPath,
+    ...objects,
+    ...boostLibs.map(name =>
+      join(boostPath, `stage/${sdk}/${arch}/libboost_${name}.a`)
+    ),
+    ...zanoLibs.map(name => join(working, `lib/lib${name}.a`))
+  ])
 
-    // Add object files to the list:
-    for (const file of await readdir(outPath)) {
-      if (file.endsWith('.o')) objects.push(join(outPath, file))
-    }
-  }
-  for (const lib of zanoLibs) {
-    await unpackLib(join(working, `lib/lib${lib}.a`), lib)
-  }
-  for (const lib of boostLibs) {
-    await unpackLib(
-      join(
-        tmpPath,
-        `zano_native_lib/_libs_ios/boost/stage/${sdk}/${arch}/libboost_${lib}.a`
-      ),
-      `boost_${lib}`
-    )
-  }
+  // Hide all symbols except the two we really want:
+  await loudExec(objcopy, [
+    objectPath,
+    '-w',
+    '-L*',
+    '-L!_zanoMethods',
+    '-L!_zanoMethodCount'
```

Comment:
Why is `-L` used instead of what is documented (`-N`)? https://llvm.org/docs/CommandGuide/llvm-objcopy.html